.gitignore

```diff
@@ -14,3 +14,5 @@ node_modules/
 .*.swp
 .cache/
 *.7z
+*.DS_Store
+plugins/librime-*
```

Comment:
install-plugins.sh drops the prefix "librime-" in the local clones.
Try if `plugins/*/` can exclude all sub-directories while still tracking changes to files.

Reply:
I pushed another commit https://github.com/rime/librime/commit/1c9c71d74343c27336ff63867a3d65fa5d383891